CODEOWNERS

```diff
@@ -1 +1,9 @@
+stable/rbac-manager @sudermanjr @robscott
+stable/aws-iam-authenticator @sudermanjr @davekonopka
+incubator/autospotting @sudermanjr @coreypobrien
+incubator/fluentd @dosullivan @coreypobrien
+incubator/helm-release-pruner @davekonopka
+incubator/imagepullsecret-deploy @crmejia
+incubator/kubebench @mjhuber
+incubator/ro-cert-manager @sudermanjr @endzyme
```

Comment:
The [Github docs](https://help.github.com/en/articles/about-code-owners) seem to suggest the opposite order: broad first followed by more specific overrides.

Reply:
I read that the opposite.  Doing this in another repo earlier seems to confirm that, but I will double check